.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: php
 
+dist: trusty
+sudo: false
```

Comment:
Not exactly related, but test on hhvm fails otherwise. This is required due to a recent Travis change.